packages/app/src/cli/services/context.test.ts

```diff
@@ -545,6 +609,74 @@ describe('ensureDeployContext', () => {
       'token',
     )
   })
+
+  test('disables developer preview if the beta is enabled in partners', async () => {
```

Comment:
do you mean, "disabled in partners"?

Reply:
no. I mean that when you run deploy with the new unified beta enabled the developer preview is automatically disabled. 
But now that I'm reading the title carefully I'm not able to understand that properly 😅

Reply:
but `enables developer preview if the beta is enabled in partners` in under different `describe` so if you run the tests you will see.

``` 
✓ src/cli/services/context.test.ts > ensureDevContext > enables developer preview if the beta is enabled in partners
✓ src/cli/services/context.test.ts > ensureDeployContext > disables developer preview if the beta is enabled in partners
```

should that title be enough to distinguish both of them?

Reply:
Ok, I will modify the title to something similar to this

```
✓ src/cli/services/context.test.ts > ensureDevContext > dev enables developer preview if the beta is enabled in partners
✓ src/cli/services/context.test.ts > ensureDeployContext > deploy disables developer preview if the beta is enabled in partners
```